### ISSUE TITLE
Fixed size on pages

### ISSUE THREAD

**Issue by Slava** [percolatestudio/mdg-localmarket#10]
Right now the app doesn't fill the whole screen, the sizes are rather hardcoded. On iPad Mini I am testing on it doesn't look like a mobile app :)

See attachment:
![img_20140903_145915](https://cloud.githubusercontent.com/assets/2608446/4142267/cdd03e76-33b6-11e4-92e8-ff8a9c5a89e8.jpg)

**Comment by domyen** [comment]
Hey @Slava, to limit scope and markup complexity our directive was to target mobile devices –specifically phones– only, thus our breakpoint/max-width is constrained to signal that intention. 

While we can remove the artificial limit, your mileage may vary.

**Comment by Slava** [comment]
I realize it was intended, not sure if supporting all sizes is in your plans?

**Comment by zol** [comment]
It wasn't but it could be. I'll raise it with Matt.

**Comment by zol** [comment]
From today's conversation, the two options:
1. Place the app in an 'iphone frame' to signify that it is mobile layout only.
2. Design/implement a full responsive layout (i.e, not just stretching out the current layout).

**Comment by domyen** [comment]
I'm much in favor of option 1. I've added iPhone chrome which makes it even more obvious what the app is intended for. 

**Option 2 considerations:**
1. Image quality is notoriously low when retrieved from recipe sources. We've managed to find the best quality image possible for each recipe however this still means many are quite low in resolution when viewed on large screens (sub 600px). On a hi-definition display (iPad, Galaxy Tablet) the images would be pixelated and blurry.
2. Creating a tablet layout means not only reconsidering markup and styling decisions, but also added complexity in the styling to achieve. i.e., +5 additional views need to be styled.  

If our purpose is to have a kitchen sink solution, creating a fully responsive solution would be great, but be wary that the markup and styling may be opaque for those who aren't _advanced_ at CSS. 

FYI the Todo's example is fully responsive.
